support/index.md

```diff
@@ -39,15 +33,15 @@ If you have a paid Docker subscription, you can get support for the following ty
       * Installation crashes
       * Failure to launch Docker Desktop on first run
    * Desktop update issues
-   * Login issues in both the command line interface and Docker Hub user interface
+   * Sign in issues in both the command line interface and Docker Hub user interface
```

Comment:
Would this work better since this is the adjective form:
```suggestion
   * Sign-in issues in both the command line interface and Docker Hub user interface
```
I always have to look this up!

Reply:
oo yup! 🔍